emspayamex/config.xml

```diff
@@ -2,7 +2,7 @@
 <module>
     <name>emspayamex</name>
     <displayName><![CDATA[ EMS Online Apple Pay]]></displayName>
-    <version><![CDATA[1.2.0]]></version>
+    <version><![CDATA[1.2.1]]></version>

```

Comment:
Why u change version there

Reply:
Because I change version everywhere. I asked that question to Claimer. He told that needs doing like this.

Reply:
But you actually don't change version everywhere, only in that payment method .xml and in afterpay.xml :)